### PR TITLE
updated swagger-ui to latest for oas3 support

### LINKED ISSUE #418: swagger-ui 3.x support
swagger-ui 3.x recently dropped. It has enough changes that the ```update.sh``` script cannot automatically upgrade it. It would be cool if we could get this in here.

### PR DESCRIPTION
**Pull request by fehguy**
Fixes #418 .

Less code!  This update uses unpkg.com to host the swagger-ui artifacts rather than use the previous `update.sh` script that lived in the vendor folder.

If maintaining a specific version of swagger-ui is desired, I can scope the version of the three assets.  Please let me know

Changes proposed in this pull request:

 - Update of the `index.html` to use the updated swagger-ui container
 - Removal of the `css`, `fonts`, `images`, `lang`, `lib` folders, as well as the `update.sh`, `o2c.html`, `*.js` files
 - Added links to `unpkg.com` for serving three static assets

### CODE REVIEW

**Comment by fehguy** [comment]
This, or something like it, will be needed for #420

**Comment by dtkav** [comment]
I definitely prefer vendoring swagger-ui (the current approach) over depending on a random 3rd party.
Also, I'd highly recommend always pinning your dependencies in general - using `latest` means that a connexion release could be broken by a swagger-ui update.

File: connexion/vendor/swagger-ui/index.html [hunk 1]
```diff
@@ -5,104 +5,38 @@
...
-
-      function log() {
-        if ('console' in window) {
-          console.log.apply(console, arguments);
-        }
-      }
-  });
-  </script>
+  <script src="https://unpkg.com/swagger-ui-dist/swagger-ui-standalone-preset.js"></script>
+  <script src="https://unpkg.com/swagger-ui-dist/swagger-ui-bundle.js"></script>
```
Comment (dtkav) [inline connexion/vendor/swagger-ui/index.html]: also pin these? looks like you used https://unpkg.com/swagger-ui-dist@3.12.1/ below.

**Comment by fehguy** [comment]
Thanks @dtkav.  I'll remove the reference to `unpkg.com` and pull in the js artifacts with an updated script.  Will update this PR shortly.

**Comment by dtkav** [comment]
Hey @fehguy - There was more context in #406 that I didn't realize. Maybe weigh in there? Also as a disclaimer - I just wanted to share my personal opinion above - the core connexion folks might have different opinions from me :smile:

**Comment by fehguy** [comment]
@hjacobs what direction do you suggest on this front?  I'll redo the PR as fit 👍

**Comment by rafaelcaricio** [comment]
I would vote for the #614 approach as optional (see my comment there).

**Comment by jmcs** [comment]
We are going with the #614 approach so I'm closing this PR. Thanks for PR anyway.

**Pull request closed without merging**